integration_tests/test_version.py

```diff
@@ -0,0 +1,5 @@
+def test_lpython_version():
+    print(__LPYTHON_VERSION__)
+    assert __LPYTHON_VERSION__ == "0.2.0-1041-g196214be3-dirty"
```

Comment:
```suggestion
from platform import python_version

def test_lpython_version():
    print(python_version())
```

Since the version changes even between commits for LPython, and it is different for CPython and LPython, we can't test it directly. But as long as it prints, that should be fine.

Reply:
> Since the version changes even between commits for LPython, and it is different for CPython and LPython, we can't test it directly. But as long as it prints, that should be fine.

Even I thought it was strange that when I was running  `./src/bin/lpython /Users/user/lpython/integration_tests/test_version.py && ./a.out ` and playing around with the assertion test the output was as expected , but it gives an error in the integration tests .

Reply:
With the new patch in, what error do you get?

The CI right now has some http errors, unrelated to this PR.